queries.sql

```diff
@@ -23,3 +23,40 @@ WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.40 AND weight_kg <= 17.30;
+
+/* Begin of a transaction */
+BEGIN;
+
```

Comment:
- [ ] Please from the project requirement, 
  - Inside a transaction:
    - Update the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon`.
    - Update the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set.
    - Commit the transaction.
    - Verify that change was made and persists after commit.

Please make sure you do that and **Take a screenshot of the results of your actions.** :pray:

Reply:
- [ ] The first reviewer asked that you update the `animals` table by setting animals whose names end in `mon` to _**Digimon**_ and the rest to **_Pokemon_**. Although your screenshot shows the results of such a transaction, the code is missing in your file. Kindly write the piece of code that sets the rest of the animals to _**Pokemon**_ and commit the transaction 👍 😎